nx-dev/ui-markdoc/src/lib/nodes/fence.schema.tsx

```diff
@@ -9,6 +9,7 @@ export const fence: Schema = {
     command: { type: 'String', default: '' },
     path: { type: 'String', default: '~/workspace' },
     process: { type: 'Boolean', render: false, default: true },
+    enableCopy: { type: 'Boolean', default: true },
```

Comment:
`isMessageBelow` seems to be missing

Reply:
no this is not something we pass in via the options, but this is being calculated in the fences.component based on the `@nrwl/` to `@nx/..` stuff